### ISSUE TITLE
Snippets don't trigger

### ISSUE THREAD

**Issue by chevdor** [karuna/tera-vscode#12]
Hello,

Is there a trick to get the snippets to trigger ?
I re-installed the plugin but none is available.

**Comment by karamfd** [comment]
I just tried using them on an HTML file and all is good. I'm using VSCode version 1.57 on Mac OS Big Sur 11.4 on an Intel based machine.

**Comment by karuna** [comment]
@chevdor is there any debugging info?

**Comment by chevdor** [comment]
I did not see much.
Do you expect a specific file extension ?

**Comment by karamfd** [comment]
@chevdor I agree that support for tera snippets in markdown can be useful. Developers can write a tutorial about using Tera on their blog site, for example.

**Comment by karuna** [comment]
Fixed
